plugins/inputs/win_perf_counters/win_perf_counters.go

```diff
@@ -149,11 +153,22 @@ type Win_PerfCounters struct {
 	Log telegraf.Logger
 
 	lastRefreshed time.Time
-	counters      []*counter
-	query         PerformanceQuery
+	queryCreator  PerformanceQueryCreator
+	hostCounters  map[string]*hostCountersInfo
+}
+
+type hostCountersInfo struct {
+	// computer name used as key and for printing
+	computer string
+	// computer name used in tag
+	computerTag string
```

Comment:
Can't `computer` be re-used for this? It seems always setting the tag (even if it is "localhost") is better than selectively doing so.

Reply:
It depends if plugin should add tag computer=localhost, when it gathers from localhost.
Until now it wasn't there, and many users will, probably, not gather from multiple hosts.

Maybe, it should add this tag only if it gathers from multiple hosts?

Reply:
I prefer always setting the tag, but it wouldn't kill me if it wasn't

Reply:
I'm really not sure what is the best way to go with. I decided to go with method with no change for users with normal usage. However, when I imagine seeing a bunch of data points gathered from multiple hosts and couple of lines show `localhost` among the real names, it doesn't look good neither. 
Maybe using real computer name, as reported in the host tag, would be the good solution, but it doesn't clearly indicate it is gathered on localhost.

Maybe we could ask Daniel for an advice, as he has a great sense for regression and good  overview of all plugins, so he may recall plugin with similar problem.

Reply:
I think we should always add the tag, for localhost though we should use `os.Hostname()` though.  Adding new tags won't break any queries so long as there isn't an existing field with the same name.

Reply:
Ok

Reply:
Yes, I think this will be okay.